pkg/hubble/parser/agent/parser_test.go

```diff
@@ -45,6 +45,7 @@ func (e *mockEndpoint) GetID() uint64           { return e.ID }
 func (e *mockEndpoint) GetOpLabels() []string   { return e.Labels }
 func (e *mockEndpoint) GetK8sPodName() string   { return e.PodName }
 func (e *mockEndpoint) GetK8sNamespace() string { return e.Namespace }
+func (e *mockEndpoint) GetID16() uint16         { return uint16(e.ID) }
```

Comment:
Could we instead update `mockEndpoint.ID` to be `uint16` and cast to `uint64` for `GetID()`?

Reply:
Done